examples/connection_pools/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "connection_pools"
+version = "0.0.0"
+workspace = "../../"
+
+[dependencies]
+rocket = { path = "../../lib" }
+rocket_codegen = { path = "../../codegen" }
+diesel = { version = "1.0", features = ["sqlite", "postgres", "r2d2"] }
```

Comment:
Maybe we should probably reexport this from `rocket_contrib` to prevent any version issues?

Reply:
No, it doesn't. That's a mistake on my part. I forgot to remove that from earlier experimentation.